app/models/organisation.rb

```diff
@@ -44,6 +44,11 @@ class Organisation < ApplicationRecord
 
   has_and_belongs_to_many :users
 
+  has_many :community_clinic_locations,
```

Comment:
We have a `has_many :clinics, through: :teams` above which will return these already plus the generic clinic. However I'm not sure it's actually used anywhere, perhaps we should change that to only return community clinics (maybe renaming it too?) and use that instead, through the team model.

Reply:
Makes sense. I've updated the implementation (see https://github.com/nhsuk/manage-vaccinations-in-schools/pull/2461/commits/ce2dbde89f3867b2a63a9143dd98318d87a56d90) - hopefully this is what you had in mind?